File: TPraticas/Aula11.a/codigofonte/LOverflow.java

public class LOverflow
 {
  public static void main(String[] args)
  {
    int importantData =1;
    int[]  buffer = new int[10];

    for (int i =0; i < 500; i++) {  
      buffer[i] = 7;
      System.out.println("i = "+i);
    }

    System.out.println("after buffer overflow ");
    System.out.println("Important data  = "+importantData);
  }
}

File: TPraticas/Aula11.a/codigofonte/LOverflow.cpp
#include <iostream>

using namespace std;

int main(void) {
    int importantData = 1;
    int  buffer[10];

    for (int i =0; i < 500; i++) {  
      buffer[i] = 7;
      cout << "i = " << i << endl;
    }

    cout << "after buffer overflow " << endl;
    cout << "Important data  = " << importantData << endl;

    return 0;
}

